src/components/IssueListItem.js

```diff
@@ -41,9 +41,11 @@ function IssueListItem({title, labels, author, opened, type, participants, comme
                   {milestone.title}
                 </span>
               )}
-              {participants && participants.nodes.map((user, key) => (
-                <img className="participants" key={key} src={user.avatarUrl} title={user.login} />
-              ))}
+              <div className="avatar-stack">
+                {participants && participants.nodes.map((user, key) => (
+                  <img className="avatar" key={key} src={user.avatarUrl} title={user.login} />
```

Comment:
if AvatalURL is empty, a placeholder image can be used.

Reply:
@sujitagarwal If `avatarUrl` is empty will not render the image at all!

Reply:
Could you please explain me the use-case? I'm trying to figure out why we're having this behavior.

Also, what's the upper limit on number of users being shown?